hrm-service/service-tests/contacts.test.js

```diff
@@ -169,11 +298,100 @@ describe('/contacts route', () => {
       expect(response.body.error).toBe('Authorization failed');
     });
 
-    test('should return 200', async () => {
-      const response = await request.get(route).set(headers);
-      expect(response.status).toBe(200);
-      expect(response.body).not.toHaveLength(0);
+    const queueName = 'queue-name';
+
+    let createdContacts = [];
+    beforeAll(async () => {
+      // Clean what's been created so far
+      await Contact.destroy(query);
+
+      // Create some contacts to work with
+      const contact = { ...contact1 };
+
+      const withQueueName = { ...contact, queueName };
+
+      const oneHourBefore = {
+        ...withQueueName,
+        timeOfContact: new Date(new Date().getTime() - 1000 * 60 * 60).toISOString(), // one hour before
```

Comment:
`subHours(new Date(), 1).toisoString()` might be more readable than hand rolling it. `date-fns` has functions for most basic date operations, and it's already a dependency so might as well use it!

Reply:
Thanks for the suggestion! :raised_hands: